recipes/bindashtree/meta.yaml

```diff
@@ -0,0 +1,32 @@
+{% set version = "0.1.2" %}
+
+package:
+  name: bindashtree
+  version: {{ version }}
+
+build:
+  number: 0
+  run_exports:
+      - {{ pin_subpackage('bindashtree', max_pin="x.x") }}
+  skip: True  # [osx]
+source:
+  url: https://github.com/jianshu93/bindashtree/archive/v{{ version }}.tar.gz
+  sha256: f22b6a10332047ee51ae98fe2c3d984ec2cfef2d0b7ce76e9b69e788173724f4
+
+requirements:
+  build:
+    - {{ compiler("cxx") }}
+    - {{ compiler('c') }}
+    - {{ compiler('rust') }}
+
+test:
+  commands:
+    - bindashtree -h
+
+about:
+  home: https://github.com/jianshu93/bindashtree
+  license: MIT
```

Comment:
Could you please add `license_file` too ?
I do not see a license in your repo - https://github.com/jianshu93/bindashtree

Please also see https://bioconda.github.io/contributor/guidelines.html#rust and add usage of `cargo-bundle-licenses`